vimrc.bundles

```diff
@@ -8,11 +8,12 @@ else
   call plug#begin('~/.vim/plugged')
 endif
 
+Plug 'google/vim-maktaba'
```

Comment:
What is `maktaba` for?

Reply:
`maktaba` is a dependency for `codefmt`. See https://github.com/google/vim-codefmt#installation